hax/hax/motr/__init__.py

```diff
@@ -282,6 +282,12 @@ def ha_obj_state(st):
                 notify_devices = False
             notes += self._generate_sub_services(note, self.consul_util,
                                                  notify_devices)
+            if (st.fid.container == ObjT.PROCESS.value
+                    and st.status == ServiceHealth.OK):
+                if self.consul_util.is_ios_service(st.fid):
+                    self.consul_util.update_sub_disks_state(st.fid, st.status)
```

Comment:
This must be moved to `_generate_sub_disks()` and must be invoked if generated disks are non-empty, actually you can iterate over list of the generated disks.

Reply:
Done